cfme/ansible/playbooks.py

```diff
@@ -53,7 +53,7 @@ class PlaybookDetailsView(PlaybookBaseView):
     class toolbar(View):   # noqa
         configuration = Dropdown("Configuration")
         policy = Dropdown(text='Policy')
-        download_button = Button(title="Download summary in PDF format")
+        download_button = Button(title="Print or export summary")
```

Comment:
Since every other view calls this `download` can you change this to `download` as well?

Reply:
There are still a few(very few) views that use `download_button` instead of `download`, I think it's better if I leave that to the FA owner, but let me know if you think otherwise, changing the name in the remaining views won't be that difficult.

Reply:
In my opinion, since this button is unused in tests, it shouldn't harm anything. And it is better to maintain consistency in the codebase. What do you think? 

Maybe ping the FA owners here so they are aware of the change.

Reply:
There you go @john-dupuy, replaced all the `download_button` to `download`!